app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  before_save :like_counter
+
+  def like_counter
+    post.update(likes_counter: post.likes.all.length)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you make this method private and we can save some code here.

  ```
  private

  def like_counter
    post.increment!(:likes_counter)
  end
  ```